winui/AvatarView/getting-started.md

```diff
@@ -1,10 +1,80 @@
 ---
 layout: post
-title: Getting Started with WinUIAvatar View control | Syncfusion
+title: Getting Started with WinUI AvatarView control | Syncfusion
 description: Learn all about getting started with the Syncfusion WinUI AvatarView (SfAvatarView) control, its elements, and more here.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Getting Started with WinUI AvatarView
\ No newline at end of file
+# Getting Started with WinUI AvatarView
+
+This section explains the steps required to add the WinUI AvatarView control and covers only the basic features needed to get started with Syncfusion `AvatarView` control.
+
```

Comment:
@MadhumithanJayapathi This section explains the steps required to add the WinUI AvatarView control with its basic features.

Reply:
@Manivannan-E Changed the explanation as mentioned.